src/middlewares/task.middleware.js

```diff
@@ -64,9 +66,12 @@ export default (store: any) => (next: any) => (action: any) => {
            * specify environment variables:
            */
 
-          const child = childProcess.spawn(instruction, args, {
+          const child = childProcess.spawn(PACKAGE_MANAGER_CMD, commandArgs, {
             cwd: projectPath,
-            shell: true,
+            env: {
+              ...commandEnv,
+              ...getPathForPlatform(),
+            },
```

Comment:
So glad we can also kill `shell: true`.

Reply:
Yes, I think we need to check the open PR #129 to simplify process killing after we merged the Windows support.